doc/backend/setting.md

```diff
@@ -378,30 +378,30 @@ To enable simple debug configuration:
     debug = True
 
 Production configuration (no reload, no debugtoolbar):
-
-    [app:tracim_web]
-    ...
-    pyramid.reload_templates = false
-    pyramid.debug_authorization = false
-    pyramid.debug_notfound = false
-    pyramid.debug_routematch = false
-
-    [DEFAULT]
-    ...
-    debug = False
-
+```
```

Comment:
According to the markdown lint rules, [code blocks should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines) and [should have a language specified](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md040---fenced-code-blocks-should-have-a-language-specified). I think here ``cfg`` or ``configuration`` should be ok

Reply:
I have created this PR too quickly :)
I have not finish to parse all documentation. I tried to finish this weekend, and thanks for all link :)